### PR TITLE
[RAPTOR-4506] skip predict if SKIP_PREDICT env var is present

### PR DESCRIPTION
**Pull request by jlpolit**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

### CODE REVIEW

File: custom_model_runner/datarobot_drum/drum/main.py [hunk 1]
```diff
@@ -115,6 +115,10 @@ def _is_arg_registered_in_subparser(sub_parser, arg):
             if sub_parser is None:
                 return
 
+            skip_predict = os.environ.get("SKIP_PREDICT")
+            if skip_predict is not None:
+                sys.argv.extend([ArgumentsOptions.SKIP_PREDICT])
+
             for env_var_key in ArgumentOptionsEnvVars.VALUE_VARS + ArgumentOptionsEnvVars.BOOL_VARS:
```
Comment (yakov-g) [inline custom_model_runner/datarobot_drum/drum/main.py:122]: Why don't you use existing boilerplate?
Add SKIP_PREDICT into ArgumentOptionsEnvVars.BOOL_VARS ...

**Review by yakov-g — CHANGES_REQUESTED** [review]

File: custom_model_runner/datarobot_drum/drum/main.py (hunk 1 above)
Comment (jlpolit) [reply custom_model_runner/datarobot_drum/drum/main.py:122]: sure

**Comment by 4dahalibut** [comment]
you don't want to always just skip predict?

**Comment by jlpolit** [comment]
> you don't want to always just skip predict?

I assumed we want to let users run prediction checks locally in drum, while giving an easy way to bypass in DR

**Review by yakov-g — APPROVED** [review]

**Review by meltzg — APPROVED** [review]

**Pull request merged**